benchmarks/test_benchmark_coo.py

```diff
@@ -3,16 +3,16 @@
 import numpy as np
 
 
-class MatrixMultiplySuite:
-    def setup(self):
-        rng = np.random.default_rng(0)
-        self.x = sparse.random((100, 100), density=0.01, random_state=rng)
-        self.y = sparse.random((100, 100), density=0.01, random_state=rng)
+def test_MatrixMultiplySuite(benchmark):
+    rng = np.random.default_rng(seed=42)
+    x = sparse.random((3, 3), density=0.01, random_state=rng)
+    y = sparse.random((3, 3), density=0.01, random_state=rng)
 
-        self.x @ self.y  # Numba compilation
+    # self.x @ self.y  # Numba compilation
```

Comment:
This should be uncommented -- The first time an operation is run after an import; some compilation happens. This gets rid of that overhead when benchmarking